docs/cli.md

```diff
@@ -18,23 +18,23 @@ at common flags that can be used with all of the subcommands.
 | Name                       | Description                                             |
 |:-------------------------- |:--------------------------------------------------------|
 | `-h (--help)`              | Display help for the specified command |
-| `--loglevel`               | Set log level of log output (0-3) (see [Log levels](#log-levels))|
-| `-U (--url)`               | URL of docker socket to use (default: `unix:///var/run/docker.sock`) |
+| `--loglevel`               | Set the level of log output (0-3) (see [Log levels](#log-levels))|
+| `-U (--url)`               | URL of the Docker socket to use (default: `unix:///var/run/docker.sock`) |
 
 #### Log levels
 
 There are four log levels:
-* `Level 0` - produces output from containers running `assemble` script and all encountered errors
-* `Level 1` - produces basic information about the executed process
-* `Level 2` - produces very detailed information about the executed process
-* `Level 3` - produces very detailed information about the executed process, alongside with listing tar contents
+* Level `0` - produces output from containers running `assemble` script and all encountered errors
+* Level `1` - produces basic information about the executed process
+* Level `2` - produces very detailed information about the executed process
+* Level `3` - produces very detailed information about the executed process, alongside with listing tar contents
```

Comment:
Level 4 produces very very detailed
Level 5 produces very very very detailed

ok not realy.
looks like level 4 doesn't add anything and level 5 adds info about credentials used to pull docker images.